### ISSUE TITLE
Please help me explain my result?

### ISSUE THREAD

**Issue by zero76114** [fchollet/keras#4662]
Dear all,
Can anyone help me explain my result below. Is it overfitting?
Train...
Train on 2086 samples, validate on 2084 samples
Epoch 1/16
15s - loss: 1.0029 - acc: 0.5834 - val_loss: 0.9636 - val_acc: 0.7202
Epoch 2/16
16s - loss: 0.8912 - acc: 0.6318 - val_loss: 0.9852 - val_acc: 0.7202
Epoch 3/16
10s - loss: 0.8368 - acc: 0.6222 - val_loss: 0.8854 - val_acc: 0.7198
Epoch 4/16
11s - loss: 0.8180 - acc: 0.6242 - val_loss: 0.7974 - val_acc: 0.7198
Epoch 5/16
11s - loss: 0.7444 - acc: 0.6453 - val_loss: 0.7618 - val_acc: 0.7188
Epoch 6/16
15s - loss: 0.6899 - acc: 0.6568 - val_loss: 0.6863 - val_acc: 0.7188
Epoch 7/16
15s - loss: 0.6539 - acc: 0.6735 - val_loss: 0.6592 - val_acc: 0.7188
Epoch 8/16
11s - loss: 0.6517 - acc: 0.6870 - val_loss: 0.6519 - val_acc: 0.7188
Epoch 9/16
11s - loss: 0.6335 - acc: 0.6889 - val_loss: 0.6284 - val_acc: 0.7188
Epoch 10/16
11s - loss: 0.6270 - acc: 0.6946 - val_loss: 0.6124 - val_acc: 0.7188
Epoch 11/16
11s - loss: 0.6072 - acc: 0.7105 - val_loss: 0.6317 - val_acc: 0.7188
Epoch 12/16
14s - loss: 0.6153 - acc: 0.7124 - val_loss: 0.6087 - val_acc: 0.7188
Epoch 13/16
16s - loss: 0.6107 - acc: 0.7167 - val_loss: 0.6095 - val_acc: 0.7188
Epoch 14/16
12s - loss: 0.6012 - acc: 0.7162 - val_loss: 0.6060 - val_acc: 0.7188
Epoch 15/16
13s - loss: 0.5984 - acc: 0.7167 - val_loss: 0.6119 - val_acc: 0.7188
Epoch 16/16
16s - loss: 0.6014 - acc: 0.7162 - val_loss: 0.6042 - val_acc: 0.7188
avg sec per epoch: 14.7203124911

**Comment by fchollet** [comment]
Not overfitting.

**Comment by zero76114** [comment]
@fchollet  Thanks for your response.
There are different accs and val_accs for each epoch. How do I know the acc or val_acc for my model as a whole? Do I average all of the epochs accs or val_accs to find the acc or val_acc of the model as a whole?
